src/settings.js

```diff
@@ -1640,6 +1640,17 @@ var SUPPORT_LONGJMP = 1;
 // selectors, instead of referring to DOM IDs.
 var DISABLE_DEPRECATED_FIND_EVENT_TARGET_BEHAVIOR = 1;
 
+// Certain browser DOM API operations, such as requesting fullscreen mode
+// transition or pointer lock require that the request originates from within
+// an user initiated event, such as mouse click or keyboard press. Refactoring
+// an application to follow this kind of program structure can be difficult, so
+// HTML5_SUPPORT_DEFERRING_USER_SENSITIVE_REQUESTS=1 flag allows transparent
+// emulation of this by deferring synchronous fullscreen mode and pointer lock
+// requests until a suitable event callback is generated. Set this to 0
+// to disable support for deferring to save code space if your application does
+// not need support for deferred calls.
```

Comment:
We now have `[link]` if its just a link time flag.

Reply:
Looks like most settings are not annotated with these yet. I'll see if I have a moment to add those annotations.